lms/djangoapps/onboarding/helpers.py

```diff
@@ -18,7 +18,6 @@
     Organization, OrganizationMetricUpdatePrompt, PartnerNetwork, OrganizationAdminHashKeys
 )
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
-from openedx.features.data_extract.models import CourseDataExtraction
```

Comment:
Let's please unregister `CourseDataExtraction` from admin as well, so that absolutely no one can interact with this table from application.

Reply:
Same for `Faq` model.

Reply:
You can not run migrate commands after unregistering an application.

Reply:
@ahmed-faraz46 wait, I thought you meant unregistering the apps, my bad.

Reply:
Done

Reply:
Nope, I mean unregister models from django admin.